tools/PreTrimmer/PreTrimmer/Steps/PreserveBlockCodeHandler.cs

```diff
@@ -0,0 +1,76 @@
+using System;
+using System.Linq;
+
+using Mono.Cecil;
+
+using Mono.Linker;
+using Mono.Linker.Steps;
+using Mono.Tuner;
+
+using Xamarin.Bundler;
+
+#nullable enable
+
+namespace Xamarin.Linker.Steps {
+	public class PreserveBlockCodeHandler : ConfigurationAwareStep {
```

Comment:
Instead of copying and modifying `tools/dotnet-linker/Steps/PreserveBlockCodeHandler.cs`, would it be possible to modify the existing file, and just use the `PRETRIM` conditional compilation symbol to modify it?

That makes it easier to see what's the difference is between the two implementations (and also avoids code duplication).